src/components/graphqlFormGenerator/components/vuetify.js

```diff
@@ -154,7 +154,7 @@ export default {
     },
     NamespaceIDGlob: {
       is: VTextField,
-      placeholder: 'name[.cycle][:status]',
+      placeholder: 'cycle[/task][:status]',
```

Comment:
Should this be?
```suggestion
      placeholder: 'cycle/task[:status]',
```

Reply:
The task is now optional e.g. `cylc hold 'workflow//*'`